Frontend/ui-driver/src/Screens/TripDetailsScreen/View.purs

```diff
@@ -300,9 +301,58 @@ tripDataView push state =
   , width MATCH_PARENT
   , orientation VERTICAL
   , gravity CENTER_VERTICAL
-  ][  tripDetailsRow push {  keyLeft : getString RIDE_TYPE, valLeft : rideType, keyRight : (getString TRIP_ID), valRight : state.data.tripId, leftClick : NoAction, rightClick : Copy, leftAsset : "", rightAsset : "ny_ic_copy", leftVisibility : true, rightVisibility : true, leftItemLeftAsset : if state.data.acRide == Just true then Just "ny_ic_ac" else Nothing},
-      tripDetailsRow push {  keyLeft : getString DISTANCE, valLeft : (state.data.distance <> " km"), keyRight : getString RIDE_TIME, valRight : tripTime, leftClick : NoAction, rightClick : NoAction, leftAsset : "", rightAsset : "", leftVisibility : true, rightVisibility : true, leftItemLeftAsset : Nothing},
-      tripDetailsRow push {  keyLeft : getString EARNINGS_PER_KM, valLeft : earningPerKm, keyRight : (getString TOLL_INCLUDED), valRight : currency <> (show state.data.tollCharge), leftClick : NoAction, rightClick : NoAction, leftAsset : "", rightAsset : "", leftVisibility : true, rightVisibility : state.data.tollCharge /= 0.0, leftItemLeftAsset : Nothing}
+  ][  tripDetailsRow push {  
+        keyLeft : getString RIDE_TYPE
+      , valLeft : rideType
+      , keyRight : getString TRIP_ID
+      , valRight : state.data.tripId, leftClick : NoAction
+      , rightClick : Copy
+      , leftAsset : ""
+      , rightAsset : "ny_ic_copy"
+      , leftVisibility : true
+      , rightVisibility : true
+      , leftItemLeftAsset : if state.data.acRide == Just true then Just "ny_ic_ac" else Nothing
+      },
+      tripDetailsRow push {  
+        keyLeft : getString DISTANCE
+      , valLeft : (state.data.distance <> " km")
+      , keyRight : getString RIDE_TIME
+      , valRight : tripTime
+      , leftClick : NoAction
+      , rightClick : NoAction
+      , leftAsset : ""
+      , rightAsset : ""
+      , leftVisibility : true
+      , rightVisibility : true
+      , leftItemLeftAsset : Nothing
+      },
+      tripDetailsRow push {  
+        keyLeft : getString EARNINGS_PER_KM
+      , valLeft : earningPerKm
+      , keyRight : getString TOLL_INCLUDED
+      , valRight : currency <> (show state.data.tollCharge)
+      , leftClick : NoAction
+      , rightClick : NoAction
+      , leftAsset : ""
+      , rightAsset : ""
+      , leftVisibility : true
+      , rightVisibility : state.data.tollCharge /= 0.0
+      , leftItemLeftAsset : Nothing
+      }
+    , tripDetailsRow push {  
+        keyLeft : getString PARKING_CHARGE
+      , valLeft : currency <> (show $ round $ state.data.parkingCharge) <>  " (" <> (getString INCLUDED) <> ")"
+      , keyRight : ""
+      , valRight : ""
+      , leftClick : NoAction
+      , rightClick : NoAction
+      , leftAsset : ""
+      , rightAsset : ""
+      , leftVisibility : state.data.parkingCharge > 0.0
```

Comment:
we can do maybe. but the thing is, if the parking Charge is 0.0 then we need to put 2 cases to don't show the value. like `isJust state.data.parkingCharge && fromMaybe 0.0 state.data.parkingCharge > 0.0` to avoid this two check only i added like this.